content/en/docs/Installation/_index.md

```diff
@@ -542,7 +546,7 @@ metadata:
 
 ## Upgrading Kyverno
 
-Upgrading Kyverno is as simple as applying the new YAML manifest, or using Helm depending on how it was installed.
+Upgrading Kyverno is as simple as applying the new YAML manifest, or using Helm depending on how it was installed. You cannot upgrade Kyverno by bumping the image tag on the Deployment as this will not affect the CRDs and other resources necessary for Kyverno's operation.
```

Comment:
If helm is not only supported install method shouldn't we remove the reference to yaml manifests ?

Reply:
Not sure I understand. The highlighted comment here refers to upgrading. From an installation standpoint, Helm is our "supported" methodology, but we still want to allow a simple YAML manifest install. I'm just calling out here that, regardless of the installation method, users cannot upgrade Kyverno by just increasing the image tag. We've seen this multiple times so I'm putting it in formal language now.

Reply:
Ok, I wondered if we wanted to remove the `applying the new YAML manifest`  and only talk about Helm, to be consistent with other sections.
Up to you, what we have here is fine too.